### ISSUE TITLE
TwoColumnPanel: Visual Heading Text is not Marked as a Heading

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1219]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/275d4118-5d34-11ee-8ab0-836408935bc1?activeTab=dt-issue&page=0&sortField=ordinal&sortDir=asc&row=3&issuesCount=&issuesCount=&filter=

## Description

Text appears and functions like a section heading but is not marked up as such.

Element description 1: 
- Items
Element location: Below the "Default" heading

Element description 2: 
- Grouped Items
Element location: Below the "With Optional Group Headings" 

## Dev Notes

We should add these sections as headings in docs.

**Comment by scurker** [comment]
This should be fixed by #1454
